pkg/globals/globals.go

```diff
@@ -41,23 +34,11 @@ type RunOpts struct {
 type GlobalOpts struct {
 	// RunOpts are inlined to allow tests to override parameters without changing ENV variables or flags
 	RunOpts
-	// HomeDir most importantly contains envoy binaries fetched from ManifestURL. Defaults to DefaultHomeDir
+	// HomeDir most importantly contains envoy binaries fetched from ManifestURL. Defaults to homedir.Dir/.getenvoy
```

Comment:
what is `homedir.Dir`?

Reply:
drifted as we no longer use that lib
```suggestion
	// HomeDir most importantly contains envoy binaries fetched from ManifestURL. Defaults to $HOME/.getenvoy
```